src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -284,7 +284,7 @@ public String getBuilderId() {
 
   @Override
   @NotNull
-  public ModuleType getModuleType() {
+  public ModuleType<?> getModuleType() {
```

Comment:
Line 205 needs `Nullable` on the first parameter.

Reply:
done